Makefile

```
@@ -112,7 +113,23 @@ installer:
 		--frontend-opt target=$@ \
 		$(COMMON_ARGS)
 	@docker load < build/$@.tar
-	@docker run --rm -v /dev:/dev -v $(PWD)/build:/out --privileged autonomy/$@:$(TAG) image -l
+
+do-image: installer
+	@docker run --rm -v /dev:/dev -v $(PWD)/build/do:/out --privileged $(DOCKER_ARGS) autonomy/installer:$(TAG) image -l \
+	-f -p digitalocean -u http://169.254.169.254/metadata/v1/user-data -e 'random.trust_cpu=on'
```

The strategy I took with AWS was to ignore the `-u` flag and set it to `-u none` and then let each platform implementation use what they have defined as a constant.
```suggestion
	-f -p digitalocean -u none -e 'random.trust_cpu=on'
```